### PR TITLE
Commit to homework_nr_11

### PR DESCRIPTION
**Pull request by Sezami**

### CODE REVIEW

File: src/main/java/homework_nr_11/Person.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        Person person8 = new Person("Iren", "Joly", 52, true);
+        Person person9 = new Person("Tatiana", "Tereza", 29, false);
+        Person person10 = new Person("Anastasia", "Volkov", 33, true);
+
+
+        List<Person> persons = new ArrayList<>(List.of(person1, person2, person3, person4, person5, person6, person7, person8, person9, person10));
+
+// Persons with age > 25 and isPayedByHour = true
+        List<Person> personsWithAgeOver25YearsAndIsPayedByHour = persons.stream()
+                .map(person -> new Person(person.name, person.surname, person.age, person.isPayedByHour))
```
Comment (gh3rtzz) [inline src/main/java/homework_nr_11/Person.java:40]: you don't need to create new objects in map. also, no need for .map() here at all.

File: src/main/java/homework_nr_11/Person.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                .filter(person -> person.age > 25 && person.isPayedByHour)
+                .toList();
+        personsWithAgeOver25YearsAndIsPayedByHour.forEach(person -> System.out.println("Persons with age > 25 and isPayedByHour: " + person.name + " " + person.surname));
+
+        System.out.println("_____________________________________________________________");
+
+// Persons with age > 30 and Surname starts with A
+        List<Person> personsWithAgeOver30YearsAndSurnameStartsWithA = persons.stream()
+                .map(person -> new Person(person.name, person.surname, person.age, person.isPayedByHour))
```
Comment (gh3rtzz) [inline src/main/java/homework_nr_11/Person.java:49]: same here

File: src/main/java/homework_nr_11/Person.java (hunk 1 above)
Comment (Sezami) [reply src/main/java/homework_nr_11/Person.java]: The corrections were done!

File: src/main/java/homework_nr_11/Person.java (hunk 2 above)
Comment (Sezami) [reply src/main/java/homework_nr_11/Person.java]: The corrections were done!

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**